test/YesSql.Tests/SqliteTests.cs

```diff
@@ -139,5 +139,50 @@ await builder
 
             await session.SaveAsync(property);
         }
+
+        [Fact]
+        public async Task ShouldDynamicIndexPropertyKeys()
+        {
+            await using (var connection = _store.Configuration.ConnectionFactory.CreateConnection())
+            {
+                await connection.OpenAsync();
+
+                await using var transaction = await connection.BeginTransactionAsync(_store.Configuration.IsolationLevel);
+                var builder = new SchemaBuilder(_store.Configuration, transaction);
+
+                await builder
+                    .DropMapIndexTableAsync<PropertyIndex>();
+
+                await builder
+                    .CreateMapIndexTableAsync<PropertyIndex>(column => column
+                        .Column<string>(nameof(PropertyIndex.Name), col => col.WithLength(4000))
+                        .Column<bool>(nameof(PropertyIndex.ForRent))
+                        .Column<bool>(nameof(PropertyIndex.IsOccupied))
+                        .Column<string>(nameof(PropertyIndex.Location), col => col.WithLength(4000))
+                    );
+
+                await builder
```

Comment:
I don't think this index will work on MySQL. Reduce the length from 4000 to like 200 on both properties to stay well under MySQL limits

Reply:
But this unit test only for Sqlite  :)

Reply:
@hyzx86 
No wonder it worked :) This test should be moved to CoreTests so it is executed on all providers.

Unit tests should pass on all database providers. It is easy to run Sqlite locally. But during deployment, we run tests on all providers.

During PR https://github.com/sebastienros/yessql/blob/main/.github/workflows/pr.yml#L61-L101

And during deployment https://github.com/sebastienros/yessql/blob/main/.github/workflows/publish.yml#L69-L100

Reply:
@hyzx86 Here is a proof that it failed MySQL ;) Fixing the issue in the next commit

![image](https://github.com/sebastienros/yessql/assets/24724371/1c1aa1bc-2fea-4848-9d30-3b23471109fb)